kolibri/core/assets/src/vue/session-nav-widget/icons/kolibri-logo.svg

```diff
@@ -0,0 +1,499 @@
+			</g>
+			<g>
+				<polygon fill="#957FB3" points="18.9,129.8 13.5,120.5 26.9,119.9 				"/>
+			</g>
+			<g>
+				<polygon fill="#8670A1" points="31,125.8 26.9,119.9 18.9,129.8 				"/>
+			</g>
+			<g>
+				<polygon fill="#AA99C9" points="25.1,137.3 18.9,129.8 31,125.8 				"/>
+			</g>
+			<g>
+				<polygon fill="#9480B3" points="36.4,131.5 31,125.8 25.1,137.3 				"/>
+			</g>
+			<g>
+				<polygon fill="#8E70A7" points="30.8,143.1 25.1,137.3 36.4,131.5 				"/>
+			</g>
+			<g>
+				<polygon fill="#A99ACA" points="41.5,135.9 36.4,131.5 30.8,143.1 				"/>
+			</g>
+			<g>
+				<polygon fill="#987FB4" points="38.6,148.6 30.8,143.1 41.4,135.9 				"/>
+			</g>
+			<g>
+				<polygon fill="#866FA0" points="47.6,140 41.4,135.9 38.6,148.6 				"/>
+			</g>
+		</g>
+		<g>
+			<g>
+				<polygon fill="#8AB7C4" points="50,46.9 55.7,42.8 58.8,54.3 				"/>
+				<g>
+					<polygon fill="#A5DDE9" points="54.2,58.3 58.8,54.3 50,46.9 					"/>
+				</g>
+				<g>
+					<polygon fill="#6A909A" points="43.1,53.3 50,46.9 54.2,58.3 					"/>
+				</g>
+				<g>
+					<polygon fill="#87BCC4" points="49.8,63.2 54.2,58.3 43.1,53.3 					"/>
+				</g>
+				<polygon fill="#A7CCE5" points="37.1,61.7 43.1,53.3 49.8,63.2 				"/>
+				<g>
+					<polygon fill="#708E9D" points="45.5,70.4 49.8,63.2 37.1,61.7 					"/>
+				</g>
+				<g>
+					<polygon fill="#A7D5E7" points="32.7,71.6 37.1,61.7 45.5,70.4 					"/>
+				</g>
+				<g>
+					<polygon fill="#8FB0C5" points="42.8,78.4 45.5,70.4 32.7,71.6 					"/>
+				</g>
+				<polygon fill="#778AA0" points="30.8,82 42.8,78.4 32.7,71.6 				"/>
+				<g>
+					<polygon fill="#A7CCE5" points="41.8,88.1 42.8,78.4 30.8,82 					"/>
+				</g>
+				<g>
+					<polygon fill="#94A7C6" points="31.1,94.3 41.8,88.1 30.8,82 					"/>
+				</g>
+				<g>
+					<polygon fill="#7E88A6" points="43.2,97.2 41.8,88.1 31.1,94.3 					"/>
+				</g>
+				<g>
+					<polygon fill="#AAC1E3" points="33.6,104.6 43.2,97.2 31.1,94.3 					"/>
+				</g>
+				<g>
+					<polygon fill="#95A5C5" points="46.1,104.3 43.2,97.2 33.6,104.6 					"/>
+				</g>
+				<g>
+					<polygon fill="#8485A8" points="38.2,113.8 33.6,104.6 46.1,104.3 					"/>
+				</g>
+				<g>
+					<polygon fill="#ABC0E3" points="49.6,110.4 46.1,104.3 38.2,113.8 					"/>
+				</g>
+				<g>
+					<polygon fill="#9A9FC7" points="43.9,121.3 38.2,113.8 49.6,110.4 					"/>
+				</g>
+				<g>
+					<polygon fill="#8585A9" points="53.9,115.3 49.6,110.4 43.9,121.3 					"/>
+				</g>
+				<g>
+					<polygon fill="#9A9FC7" points="50.1,126.9 43.9,121.3 53.9,115.3 					"/>
+				</g>
+				<g>
+					<polygon fill="#99A2C7" points="59.5,119.8 53.9,115.3 50.1,126.9 					"/>
+				</g>
+				<g>
+					<polygon fill="#AEB8DE" points="56.7,131.4 50.1,126.9 59.5,119.8 					"/>
+				</g>
+				<g>
+					<polygon fill="#ABBFE3" points="65.7,123.2 59.5,119.8 56.7,131.4 					"/>
+				</g>
+			</g>
+			<g>
+				<polygon fill="#6A909A" points="58.8,54.3 64.5,51.1 55.7,42.8 				"/>
+			</g>
+		</g>
+	</g>
+</g>
+</svg>

```

Comment:
This is a pretty complex SVG - might be smaller to include as a PNG? We can look into it later